build/testing/integration/api/api.go

```diff
@@ -574,7 +586,7 @@ func API(t *testing.T, ctx context.Context, client sdk.SDK, opts integration.Tes
 		})
 		require.NoError(t, err)
 
-		if !namespaceIsDefault(namespace) {
+		if !namespaceIsDefault(namespace) && authConfig != integration.AuthNamespaced {
```

Comment:
I am about to push this change. These should be run under the namespaced condition.
```suggestion
		if !namespaceIsDefault(namespace) {
```

Reply:
As with the other, I see why this is. This is attempting cross namespace persistence, which isn't possible for different reasons with namespaced tokens.